Tests/LibWeb/Text/expected/HTML/WindowOrWorkerGlobalScope-reportError.txt

```diff
@@ -0,0 +1,5 @@
+message = [object Error]
```

Comment:
Hmm, message is not properly formatted using JS::Value::to_string_without_side_effects`, which was suggested in https://github.com/LadybirdBrowser/ladybird/pull/392#discussion_r1664402543

Reply:
Can we use `JS::Value::to_string` instead here?

Reply:
@tcl3 Added a test but the behaviour was not expected. How can we format the error properly here without causing any side effects?